.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: php
+
+sudo: false
+
+env:
+  global:
+    - GITHUB_TOKEN=foo
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 7
+    - php: hhvm
+ allow_failures:
+    - php: hhvm
```

Comment:
The build for `hhvm` failed, so I added it here.

See https://travis-ci.org/facebook/facebook-instant-articles-sdk-php/builds/124255378.